lib/simplifyExpression/stepThrough.js

```diff
@@ -69,6 +70,9 @@ function step(node) {
   const simplificationTreeSearches = [
     // Basic simplifications that we always try first e.g. (...)^0 => 1
     basicsSearch,
+    // Substitute symbols present in the optional scope object with
+    // their respective expressions.
```

Comment:
can you add an example similar to the other simplification searches here?

Reply:
looks good to me! awesome
I think you can take out the comma after the `e.g.` though